BlogTemplate.Tests/Pages/EditTests.cs

```diff
@@ -43,5 +43,101 @@ public void UpdatePostP2P_TitleIsUpdated_UpdateSlug()
 
             Assert.Equal("Edited-Title", post.Slug);
         }
+        [Fact]
+        public void UpdatePostD2P_TitleIsUpdated_UpdateSlug()
+        {
+            IFileSystem testFileSystem = new FakeFileSystem();
+            BlogDataStore testDataStore = new BlogDataStore(new FakeFileSystem());
+            SlugGenerator slugGenerator = new SlugGenerator(testDataStore);
+            ExcerptGenerator excerptGenerator = new ExcerptGenerator();
+
+            Post post = new Post()
+            {
+                Title = "Title",
+                Slug = "Title",
+                Body = "Body",
+                Excerpt = "Excerpt",
+                IsPublic = false,
+            };
+
+            testDataStore.SavePost(post);
+
+            EditModel testEditModel = new EditModel(testDataStore, slugGenerator, excerptGenerator);
+            testEditModel.PageContext = new PageContext();
+            testEditModel.OnGet(post.Id);

```

Comment:
You don't need to call this.

Reply:
The test fails when I remove the testEditModel.OnGet(post.Id) line.